include/operations/OpType.hpp

```diff
@@ -163,6 +166,8 @@ inline std::string shortName(const OpType& opType) {
     return "sw";
   case iSWAP:
     return "isw";
+  case iSWAPdg:
+    return "iswd";
```

Comment:
As implied by the docstring of this method, short names are supposed to be 3 characters at most.
Otherwise, formatting during printing is all kinds of messed up 🙃 
What about using `"is"` for the iSWAP gate ad `"isd"` for the inverse for consistency reasons?

Reply:
I think we should still keep `"isw"` for iSWAP and `"isd"` for the inverse. The argument would be not to be mistaken with inverse S gate, but in the end, it's not that big of a deal. (At the moment it is `isw` and `isd` but I can change it if you think we should stick with the shorter one)